fractions/logstash/src/main/java/org/wildfly/swarm/logstash/runtime/LogstashCustomizer.java

```diff
@@ -54,11 +57,19 @@ public void customize() {
                     .namedFormatter("logstash")
                     .properties(handlerProps);
 
+            List<String> rootLoggerHandlers = new ArrayList<>();
+
+            RootLogger existingRootLogger = this.logging.subresources().rootLogger();
+            if (existingRootLogger != null && !existingRootLogger.handlers().isEmpty()) {
+                rootLoggerHandlers.addAll(existingRootLogger.handlers());
+            }
+            rootLoggerHandlers.add(logstashHandler.getKey());
+
             this.logging
                     .customFormatter("logstash", "org.jboss.logmanager.ext", "org.jboss.logmanager.ext.formatters.LogstashFormatter",
                                      this.logstash.formatterProperties())
                     .customHandler(logstashHandler)
-                    .rootLogger(this.logstash.level(), logstashHandler.getKey());
+                    .rootLogger(this.logstash.level(), rootLoggerHandlers.toArray(new String[rootLoggerHandlers.size()]));
```

Comment:
AS you said on IRC, I was trying to simplify this p-r  like this:

``` java
this.logging
   [...]
  .subresources()
  .rootLogger()
  .handler(logstashHandler.getKey());
```

but, org.wildfly.swarm.config.logging.RootLogger.handlers could be `java.util.Arrays$ArrayList` which cannot be added element(thrown UnsupportedOperationException).

``` java
/**
 * The handlers associated with the root logger.
 */
@SuppressWarnings("unchecked")
public T handlers(String... args) {
  handlers(Arrays.asList(args));
  return (T) this;
}
```
> If that helps, when specified `swarm.logging.root-logger.handlers` with yml, handlers will be overwritten with java.util.ArrayList.

It seems that we need to re-create it to set up root-logger with LoggingFraction.rootLoggerLevel level, String... handlers) like my above comment for now.

Should it be fixed in config-api?